src/scalarstats.jl

```diff
@@ -153,9 +181,21 @@ end
 #
 #############################
 
+"""
+    percentile(v, p) -> Float
+
+Return the `p`th percentile of an array `v`.
```

Comment:
Could continue saying "real-valued array" here and below as you did in other places.